### DISCUSSION TITLE
Should we include in the Meeting Notes the names of folks who attended the meeting?

### DISCUSSION THREAD

**Discussion by sereedmedia** [WordPress/Marketing-Team#206] [Polls]
As we work through out meeting notes revision process, we've come upon an issue that has a lot of viewpoints: tracking attendance.  I thought I'd try a poll to get people's quick responses. 

You can compare two recent versions. For the [Feb 28 notes](https://make.wordpress.org/marketing/2023/03/13/notes-global-marketing-team-meeting-28-february-2023/), we cut-and-pasted the attendance thread and formatted it. It took about 3 minutes. For this week's [March 7 notes](https://make.wordpress.org/marketing/2023/03/14/notes-global-marketing-team-meeting-7-march-2023/), we linked back to the Slack attendance thread. It took about 3 seconds.

I've summarized the basic pros and cons I've heard from folks here. If you have more thoughts on this specific issue, please comment here!

## Question
Should we cite in the meeting notes the names of the folks who attend (sync or async) the weekly marketing meeting?

### Pros
Contributors get mentioned and tagged, which allows for contribution to be logged on the team P2 and individual profiles. This also provides another way for folks to stay informed, as when they get pinged on the notes, it may encourage them to review the notes and stay engaged. Additionally, the P2 is open to the public, whereas folks who don't have Slack login cannot see a Slack link. 

### Cons
Tagging people adds to the time it takes to do the notes. It is not clear if there is positive or negative value for contributors in being tagged. It can clutter up the experience for readers.

**Comment by jennimckinnon** [comment]
I think maybe we could log attendance, just perhaps not tag people so it doesn't clutter people's feeds. That way, contributions can be noted and it can still encourage people to engage, without it being overwhelming.

If this isn't something that the majority is interested in, then my suggestion switches from "Neither yes or no" to a "Yes."

**Comment by meherbala** [comment]
I think it is good to link it to the attendee's thread because if someone is writing the notes after the meeting and it gets published, there is a chance we may miss the names of people who work async.

**Comment by sereedmedia** [comment]
**Thank you so much** for the feedback so far! 

For some background, we (team reps) are currently working with the goal to publish the notes the day before the next weekly meeting. 

To clarify with an example, using Week A and Week B as variables for any two consecutive meetings:

- The notes from Week A are published the day prior to Week B's meeting. 
- This allows six days for folks to review, comment, and discuss async on Week A's meeting threads. 
- If a thread from Week A has any discussion, Week A's notes provides a summary. 
- Prior to participating in Week B's meeting, contributors can review Week A's notes to get caught up on any discussions they missed.

**Comment by santanainniss** [comment]
As we continue the effort to streamline notes and speed up that process, I do not favor adding individual links, handles, or names to the attendance thread in the notes. I am in favor of linking to the attendance thread.

**Comment by sereedmedia** [comment]
Well, 60% of those voting said no, so we can close this poll and move on!!
